### PR TITLE
feat(tabset): fixes #266

### LINKED ISSUE #266: Tabset: support nav-pils
Trivial feature: it should be as easy as introducing a new input ("type (Defaults: tabs) - Navigation type. Possible values are 'tabs' and 'pills'") and swapping  CSS class from the default `nav-tabs` to `nav-pils`.

Tests are mandatory.

### PR DESCRIPTION
**Pull request by sendilkumarn**
Adding tabs / pills option and their respective test cases.

### CODE REVIEW

File: src/tabset/tabset.ts [hunk 1]
```diff
@@ -31,7 +31,7 @@ export class NgbTab {
 @Component({
   selector: 'ngb-tabset',
   template: `
-    <ul class="nav nav-tabs" role="tablist">
+    <ul [class]="'nav nav-'+type" role="tablist">
```
Comment (icfantv) [inline src/tabset/tabset.ts]: spacing around operator

Comment (sendilkumarn) [inline src/tabset/tabset.ts]: wondering why it was not captured in check-format 👍

**Comment by icfantv** [comment]
@sendilkumarn just the spacing around the operator and this LGTM.  @pkozlowski-opensource || @Foxandxss?

File: src/tabset/tabset.ts (hunk 1 above)
Comment (icfantv) [inline src/tabset/tabset.ts]: suffice it to say that the hate felt towards the clang formatter is quite deep and very passionate.

**Comment by pkozlowski-opensource** [comment]
Merged as 1b05d447ee9e66e421d10ad392578565151b7e90.

@sendilkumarn in the future please:
* squash all the commits into one (there should be only one commit in a PR)
* add `Fixes #266` in the commit body not a title (not a first line of the commit)

Thnx!

**Pull request closed without merging**